Ingress/images/nginx-alpine/rc.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginxalpinesvc
+  labels:
+    app: nginxalpine
+spec:
+  type: NodePort
+  ports:
+  - port: 80
+    protocol: TCP
+    name: http
+  selector:
+    app: nginxalpine
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: nginxalpine
+spec:
+  replicas: 1
+  selector:
+    app: nginxalpine
+  template:
+    metadata:
+      labels:
+        app: nginxalpine
+        name: frontend
+    spec:
+      containers:
+      - name: nginxalpine
```

Comment:
actually lets just call this nginx-slim? that's the real goal right. if anyone really cares what it's based on they can check the docker file.